protocol/urls/urls.go

```diff
@@ -293,3 +293,11 @@ func GetLinkPreviewData(link string) (previewData LinkPreviewData, err error) {
 		return previewData, fmt.Errorf("link %s isn't whitelisted. Hostname - %s", link, u.Hostname())
 	}
 }
+
+func EncodeDataURL(data []byte) (string, error) {
+	return encodeDataURL(data)
+}
+
+func DecodeDataURL(data string) ([]byte, error) {
```

Comment:
Unit tests for the public functions `EncodeDataURL` and `DecodeDataURL` seem appropriate.

Reply:
Not wanting to dogpile you @borismelnik, I noticed your comment https://github.com/status-im/status-go/pull/3650#issuecomment-1599276139 , but we should never introduce code into this repo without proving that it does what it is expected to do.

Reply:
okay! I will add the tests in this PR

Reply:
Thank you very much, I really appreciate that 🙏